### PR TITLE
toggle powerdns forwarding

### PR DESCRIPTION
**Pull request by justinjpacheco**
- added support to toggle powerdns forwarding for the local zone

### CODE REVIEW

**Review by mboruta1 — DISMISSED** [review]

**Review by comay — APPROVED** [review]

**Review by tj90241 — COMMENTED** [review]
Should the `bcpc::powerdns` chef recipe also be skipped if this is false?

I don't think the enabled flag propagates down to the Chef recipe at present.

File: ansible/playbooks/roles/common/defaults/main/base.yml [hunk 1]
```diff
@@ -295,3 +295,9 @@ system_parameters_net.nf_conntrack_max: 262144  # noqa var-naming
```
Comment (mboruta1) [inline ansible/playbooks/roles/common/defaults/main/base.yml:295]: Shouldnt we also change unbound such that it does not attempt to create DNS zones if powerdns is disabled? https://github.com/bloomberg/chef-bcpc/blob/development/chef/cookbooks/bcpc/recipes/unbound.rb#L29

Comment (justinjpacheco) [reply ansible/playbooks/roles/common/defaults/main/base.yml:295]: yep, good point, @mboruta1

Comment (justinjpacheco) [reply ansible/playbooks/roles/common/defaults/main/base.yml:295]: done

**Comment by justinjpacheco** [comment]
> Should the `bcpc::powerdns` chef recipe also be skipped if this is false?
> 
> I don't think the enabled flag propagates down to the Chef recipe at present.

@tj90241 that was added here: https://github.com/bloomberg/chef-bcpc/commit/1ec6ea15a39ab55d3864491e64333a9df6473035

**Review by mboruta1 — DISMISSED** [review]

**Review by mboruta1 — APPROVED** [review]

**Pull request merged**